.travis.yml

```diff
@@ -16,48 +16,32 @@ addons:
 env:
   global:
     - TESTS_LAMINAS_LDAP_ONLINE_ENABLED=true
-    - COMPOSER_ARGS="--no-interaction"
+    - COMPOSER_ARGS="--no-interaction --ignore-platform-reqs"
```

Comment:
Please revert this change and it to the related PHP version. You can compare with the Travis configuration of laminas-validator: https://github.com/laminas/laminas-validator/blob/23623b4756eab850829e5b5f6205eeaf1303e959/.travis.yml#L7-L38

Reply:
done - I kept `--no-interaction` in the PHP 8.0 value also, that seems "a good thing" to make sure that composer does not accidentally ask questions.